src/integration/control/configureIndex.test.ts

```diff
@@ -136,17 +136,52 @@ describe('configure index', () => {
       });
     });
 
+    test('Add index tag to a serverless index', async () => {
+      const description = await pinecone.describeIndex(serverlessIndexName);
+      expect(description.tags).toEqual({
+        project: 'pinecone-integration-tests',
+      });
+
+      await pinecone.configureIndex(serverlessIndexName, {
+        tags: { testTag: 'testValue' },
+      });
+      const description2 = await pinecone.describeIndex(serverlessIndexName);
+      expect(description2.tags).toEqual({
+        project: 'pinecone-integration-tests',
+        testTag: 'testValue',
+      });
+    });
+
     test('Remove index tag from serverless index', async () => {
       const description = await pinecone.describeIndex(serverlessIndexName);
       expect(description.tags).toEqual({
         project: 'pinecone-integration-tests',
+        testTag: 'testValue',
       });
 
       await pinecone.configureIndex(serverlessIndexName, {
         tags: { project: '' },
       });
       const description2 = await pinecone.describeIndex(serverlessIndexName);
-      expect(description2.tags).toBeUndefined();
+      if (description2.tags != null) {
```

Comment:
If `undefined != null`, does this block still run? 

Separately, are tags always returned, even if they're empty? I guess that would be my expectation as a user of the SDK.

Reply:
Question 1: `undefined` does equal `null`.

Question2 : Yes!